src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -82,6 +82,24 @@
     }
 @end
 
+@snippet pagedMethodCallSampleCode(apiMethod)
```

Comment:
If these new functions are specific to standalone samples, move them to `standalone_sample.py`. If they're used by both in-code samples and standalone samples (which may well mean they are only temporary stubs for the standalone samples until we flesh them out), leave them here.

Reply:
Ack